File: java/src/test/java/org/example/AlarmTest.java
package org.example;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertFalse;

public class AlarmTest {

    @Test
    public void alarmIsOffByDefault()
    {
        Alarm alarm = new Alarm();
        Assertions.assertFalse(alarm.isAlarmOn());
    }

}

File: kotlin/src/test/kotlin/org/example/AlarmTest.kt
package org.example

import org.junit.jupiter.api.Assertions.assertFalse
import org.junit.jupiter.api.Test

class AlarmTest {

    @Test
    fun alarmIsOffByDefault() {
        val alarm = Alarm()
        assertFalse(alarm.isAlarmOn())
    }

}